xbmc/storage/MediaManager.cpp

```diff
@@ -760,3 +764,42 @@ void CMediaManager::RemoveDiscInfo(const std::string& devicePath)
   if (it != m_mapDiscInfo.end())
     m_mapDiscInfo.erase(it);
 }
+
+bool CMediaManager::playStubFile(const CFileItem& item)
+{
+  // Make sure we're actually dealing with a Disc Stub
+  if (!item.IsDiscStub())
+    return false;
+
```

Comment:
can be ommited because you already check when calling

Reply:
Removed this check